src/axes/categoryAxis.ts

```diff
@@ -142,6 +143,12 @@ export class Category extends Axis<string> {
     return this.getDownsampleInfo().domain;
   }
 
+  protected _sizeFromOffer(availableWidth: number, availableHeight: number) {
```

Comment:
Spoke offline - without this hack, category axis layout gets borked. In the interest of speed we're going to just keep this in for now